Cargo.toml

```diff
@@ -40,6 +40,7 @@ trufflesuite_ganachecli = []
 victoria_metrics = []
 zookeeper = []
 cockroach_db = []
+kwok_cluster = []
```

Comment:
I think we should name the feature (and the module folder) `kwok`, because that's the official term as far as I understand.

Reply:
They have two container images. The cluster version (`registry.k8s.io/kwok/cluster:v0.5.2-k8s.v<k8s-version-here>`) is an `all-in-one` that runs a cluster without extra config. The other image (`registry.k8s.io/kwok/kwok:v0.5.2`) is basically to run inside a kubernetes cluster or pointing to an existing one.
I would suggest to keep the names similar with their releases so that others could distinguish them.

Reply:
Actually, we can take a look at redis module:
it has standalone and cluster version, but they are under the same feature and module

So I'd suggest to use `kwok` feature here, but keep fully-qualified name of the image, e.g `KwokCluster`

Reply:
Done.